docs/components/modeler/bpmn/user-tasks/user-tasks.md

```diff
@@ -55,6 +55,29 @@ The unique identifier depends on the authentication method used to login to Task
 For example, say you log into Tasklist using Camunda Platform 8 login with email using your email address `foo@bar.com`. Every time a user task activates with `assignee` set to value `foo@bar.com`, Tasklist automatically assigns it to you. You'll be able to find your new task under the task dropdown option `Claimed by me`.
 :::
 
+## Scheduling
+
+User tasks support specifying a task schedule using the `zeebe:taskSchedule` extension element.
+This can be used to define **when** users interact with a given task. One or both of the following
+attributes can be specified simultaneously:
+
+- `dueDate`: Specifies the due date of the user task.
+- `followUpDate`: Specifies the follow-up date of the user task.
+
+You can define the due date and follow-up date as static values (e.g. `2023-02-28T13:13:10+02:00`), but you can also use
+[expressions](/components/concepts/expressions.md) (e.g. `= schedule.dueDate` and `= now() + duration("PT15S")`). The
+expressions are evaluated on activating the user task and must result in a `string` conforming to an ISO 8601 combined
+date and time representation.
+
+:::note
+An ISO 8601 combined date and time representation must contain timezone information, either `Z` for UTC or a zone offset.
+Optionally, it can contain a zone id. The following examples represent valid ISO 8601 `strings`:
+
+- `2019-10-01T12:00:00Z` - UTC time
+- `2019-10-02T08:09:40+02:00` - UTC plus two hours zone offset
+- `2019-10-02T08:09:40+02:00[Europe/Berlin]` - UTC plus two hours zone offset at Berlin
+  :::
```

Comment:
This paragraph is duplicated from the [Timer docs](https://docs.camunda.io/docs/next/components/modeler/bpmn/timer-events/#time-date). Might be good to get some hints from a technical writer.

Reply:
I don't have any technical adjustments for this commit -- but taking a closer look at this, are you worried this section may not be as understandable to the user?

Reply:
Hey @christinaausley, I'm worried about the duplication of the content and future maintainability. I think it should be pretty clear to the user.

Are there any guidelines on how to avoid duplicated content for the docs? I couldn't find any.

Reply:
Ah, this is a great question.

@akeller and I had this discussion some time ago when we found ourselves copying over instructions to create a cluster. For maintainability purposes, I ended up creating a React component for this we could call anytime:

<img width="1312" alt="Screen Shot 2023-03-10 at 9 25 44 AM" src="https://user-images.githubusercontent.com/84338309/224369478-cee99244-2c73-45b1-b31c-8e1674d54042.png">

I find this useful when there are several documents which need to refer to the same concept.

Reply:
@christinaausley that's great! I used it for the ISO 8601 format. Let me know if it's all good.